config/k8s/config.yaml

```diff
@@ -1,9 +1,9 @@
-sharedSecretFile: $(CONFIG_MOUNT_PATH)/shared-secret
-baseUrl: $(OAUTH_HOST)
+sharedSecret: yaddayadda123$@#**
 serviceProviders:
 - type: GitHub
   clientId: "123"
   clientSecret: "42"
 - type: Quay
   clientId: "456"
   clientSecret: "54"
+baseUrl: http://oauth.192.168.64.46.nip.io
```

Comment:
Is this dummy value required? Wouldn't something like `<REPLACE_ME_WITH_A_REAL_VALUE>` be more descriptive?

Also, we should be using `https` scheme here, IMHO.

Reply:
replaced with placeholder